NEWS

```diff
@@ -1,3 +1,23 @@
+openscap-1.3.2                                                  13-01-2020
+  - New features
+    - Offline mode support for environmentvariable58 probe
+  - Maintenance, bug fixes
+    - Improved support of multi-check rules (report, remediations, console output)
+    - Improved HTML report look and feel, including printed version
+    - Less clutter in verbose mode output; some warnings and errors demoted to verbose mode levels
+    - Probe rpmverifyfile use and return canonical paths
+    - The oscap-docker wrapper is available without Atomic
```

Comment:
Maybe this could be promoted to a new feature?

Reply:
Isn't it an implementation detail? Is there any new feature coming with this change?

Reply:
Yes and no, due to this change, Ubuntu users can now scan the Docker containers, which they could not do before, because Atomic isn't available on Ubuntu.